Configuration/TCA/tx_seminars_seminars.php

```diff
@@ -162,12 +161,12 @@
             'exclude' => 1,
             'label' => 'LLL:EXT:seminars/Resources/Private/Language/locallang_db.xlf:tx_seminars_seminars.event_type',
             'config' => [
-                'type' => 'select',
+                'type' => \OliverKlee\Seminars\BackEnd\TceForms::getSelectType(),
```

Comment:
Could you please undo these changes? This method is gone from master. Thanks!

Reply:
Ok... I'll undo these changes and make an new pullrequest for the Version 2.1.1.
Where can I find this method? 
Thanks!

Reply:
The method is no longer there, i.e., I've completely removed it.